sdk/core/src/debug/java/com/klaviyo/core/Console.kt

```diff
@@ -0,0 +1,19 @@
+package com.klaviyo.core
+
+import android.util.Log
+
+/**
+ * Android Log output wrapper
+ */
+object Console {
+    enum class Level {
```

Comment:
Neat!

Reply:
Oh btw swift has a way to capture file and line number not sure if that’s possible in kotlin but might be nice to have!

Reply:
I do capture classname as the "tag". I'm not sure if i could get line number, but if we're logging an exception it should log the whole stack trace. For info/debug i didn't think line number was especially necessary, you can pretty much tell where it came from just from the message string anyway

Reply:
looks like you do use the stacktrace object below so youll have the line number, file name, class name and everything

Reply:
ya, I adapted that from timber code like we talked about during grooming, just hadn't explored what other properties are in there. makes sense line number would be

Reply:
Copied over the line number thing from kenny's utils repo